packages/modules/viewer-react/src/services/iModel/IModelService.ts

```diff
@@ -19,20 +17,15 @@ const getVersion = async (
   if (changeSetId) {
     return IModelVersion.asOfChangeSet(changeSetId);
   }
-  const token = await IModelApp.authorizationClient?.getAccessToken();
-  if (token) {
+
+  const accessToken = await IModelApp.authorizationClient?.getAccessToken();
+  if (accessToken && IModelApp.hubAccess) {
     try {
-      const hubClient = new IModelHubClient(); // TODO 3.0 - this needs to be configurable to support iTwin Stack
-      const namedVersions = await hubClient.versions.get(
-        token,
+      const changeset = await IModelApp.hubAccess.getChangesetFromNamedVersion({
```

Comment:
wouldnt this require a named version name? I thought we were going to query for that on the client for now?

Reply:
if the named version param is undefined, itll fetch the latest named ver. And i was able to query it in on the client (in the FrontendHubAccess client)